README.md

```diff
@@ -1,2 +1,51 @@
 # test
 
+## Overview 
+
+This is the home to automated end-to-end tests for the Crossplane Infrastructure. It contains logic and configuration files for the testing of Crossplane and its stack. 
+
+It uses Go and Github actions primarily to run tests and automated workflows. To create a new workflow place a workflow file in the `.github` directory of the repository. These workflows install crossplane into a `kind` cluster to run different test scenarios. 
+
+You can have a look at actions tab to see all the automated workflow runs and their status.
+
```

Comment:
@hasheddan Will it be a good idea to include the current automated workflows? Maybe something like this:
Currently, this repo holds the following tests that run on a daily basis:
- e2e tests against the latest stable release of Crossplane
- provider upgrade tests
- core crossplane upgrade test

Reply:
That sounds good, maybe link to the workflow file for each of them.